src/components/pages/index.js

```diff
@@ -1,12 +1,14 @@
-//> Components
```

Comment:
There are no pages yet. Please remove the folder `src/components/pages` and re-add it as soon as you create pages.

Reply:
We preferred to create the structure with this branch, to make further contributing easier.